docs/quickstarts/expo.mdx

```diff
@@ -357,16 +359,17 @@ export default function SignUpScreen() {
 
 #### `Sign-in page`
 
-The following example creates a sign-in page that allows users to sign in using email address and password, or navigate to the sign-up page.
+The following **example** creates a sign-in page that allows users to sign in using email address and password, or navigate to the sign-up page.
```

Comment:
Is there a reason you wanted to add bolding to "example" - we don't do that anywhere else in the docs, so I just want to see if it's a change worth inciting across the docs for consistency

```suggestion
The following example creates a sign-in page that allows users to sign in using email address and password, or navigate to the sign-up page.
```

Reply:
I definitely did this by accident lol